lib/consumers/archive-view.js

```diff
@@ -1,5 +1,6 @@
 "use strict";
 
+const pathModule = require("path");
```

Comment:
Could you please use this instead?

~~~js
const {sep} = require("path");
~~~